alfasim_sdk/variables.py

```diff
@@ -0,0 +1,47 @@
+import numbers
+from enum import Enum
+from typing import Optional
+
+import attr
+from attr import attrib
+from attr.validators import instance_of, optional
+from barril.units import UnitDatabase
+
+from alfasim_sdk._validators import check_for_valid_unit, check_string_is_not_empty
+
+
+class Visibility(Enum):
```

Comment:
Will those new classes have documentation? Or the user should look in an external document what does each one of those mean?

Reply:
Both. 😁 

We need to include docstrings on those classes, which will be inserted in the manual documentation using `autodoc`.